service/openapi.yaml

```diff
@@ -11,9 +11,11 @@ security:
   - ApiKeyAuth: []
   - BearerAuth: []
 paths:
-  /ofrep/v1/evaluate/{key}:
+  /ofrep/v1/flags/{key}:
```

Comment:
The proposal in the comment was to use `/ofrep/v1/evaluate/flags.`
I've already edited the other branch with the proposal.

Reply:
I'm fine with that too 👍

Reply:
How necessary do we think `/ofrep/` is?

Reply:
You have a discussion about that in the other PR.
We added /ofrep/ to avoid any collision with existing v1 that can exist in other systems